modules/aspect_bazel_lib/1.27.0/MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+"aspect-build/bazel-lib"
+
+module(
+    name = "aspect_bazel_lib",
+    compatibility_level = 1,
+    version = "1.27.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.4.1", dev_dependency = True)
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.5.0")
+
+ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
+
+use_repo(ext, "copy_directory_toolchains")
```

Comment:
You can pass multiple positional arguments to a single use_repo call.

Reply:
Ah yes, will fix that in bazel-lib. Thanks.